client/app/lib/components/redux-form/MultiSelect.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChipInput from 'material-ui-chip-input';
+import createComponent from './createComponent';
+
+export const optionShape = PropTypes.shape({
+  id: PropTypes.number,
+  text: PropTypes.string,
+});
+
+const propTypes = {
+  onChange: PropTypes.func.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
+  value: PropTypes.arrayOf(PropTypes.number),
+  options: PropTypes.arrayOf(optionShape),
+  error: PropTypes.string,
+  isLoading: PropTypes.bool,
+};
+
+const styles = {
+  menuStyle: {
+    maxHeight: '80vh',
+    overflowY: 'auto',
+  },
+};
+
+const MultiSelect = (props) => {
+  const { label, value, options, error, isLoading, onChange } = props;
+
+  const seletedOptions = (value || []).map(v => options.find(o => o.id === v));
+
+  return (
+    <React.Fragment>
+      <ChipInput
+        value={seletedOptions}
+        dataSource={options}
+        dataSourceConfig={{ value: 'id', text: 'title' }}
+        onBeforeRequestAdd={(chip) => {
+          // don't allow adding of arbitrary values by typing
+          if (typeof chip === 'string' || chip instanceof String) {
```

Comment:
Just to confirm, so if `ChipInput` detects that you entered a string that doesn't below to the `options` (ie. `dataSource`), it will trigger this callback with the `chip` variable being a string?

Reply:
Any input via the Enter key, even one that is a valid option, will trigger this callback with a string variable. This check completely disables adding chips via pressing Enter.

Right now on master, if you type a valid option and hit enter, the added chip will look correct but the stored value is actually the entered value instead of the skill ID.